test/horreum_client_it.py

```diff
@@ -0,0 +1,101 @@
+import httpx
+import pytest
+from kiota_abstractions.authentication import BaseBearerTokenAuthenticationProvider
+from kiota_abstractions.base_request_configuration import RequestConfiguration
+from kiota_abstractions.headers_collection import HeadersCollection
+from kiota_abstractions.method import Method
+from kiota_abstractions.request_information import RequestInformation
+
+from horreum.horreum_client import new_horreum_client, HorreumClient
+from horreum.raw_client.api.test.test_request_builder import TestRequestBuilder
+from horreum.raw_client.models.protected_type_access import ProtectedType_access
+from horreum.raw_client.models.test import Test
+
+username = "user"
+password = "secret"
+
+
+@pytest.fixture()
+async def anonymous_client_without_check() -> HorreumClient:
+    print("Setting up anonymous client")
+    return await new_horreum_client(base_url="http://localhost:8080")
+
+
+@pytest.fixture()
+async def anonymous_client() -> HorreumClient:
+    print("Setting up anonymous client")
+    client = await new_horreum_client(base_url="http://localhost:8080")
+    try:
+        await client.raw_client.api.config.version.get()
+    except httpx.ConnectError:
+        pytest.fail("Unable to fetch Horreum version, is Horreum running in the background?")
+    return client
+
+
+@pytest.fixture()
+async def authenticated_client() -> HorreumClient:
+    print("Setting up authenticated client")
+    client = await new_horreum_client(base_url="http://localhost:8080", username=username, password=password)
+    try:
+        await client.raw_client.api.config.version.get()
+    except httpx.ConnectError:
+        pytest.fail("Unable to fetch Horreum version, is Horreum running in the background?")
+    return client
+
+
+def test_check_client_version(anonymous_client_without_check: HorreumClient):
+    version = anonymous_client_without_check.version()
+    # TODO: we could load the toml and check the versions match
+    assert version != ""
```

Comment:
This seems redundant (down to the `TODO`...) with `test/horreum_client_test.py`...did I miss something?

Reply:
yeah just leftover, I moved that test to `test/horreum_client_test.py` but I forgot to remove from this file.